config/ROOT.scons

```diff
@@ -28,7 +28,7 @@ rootdict = Builder(generator = generate_action,
 rootenv = Environment(ENV={'PATH' : os.environ['PATH']})
 rootenv.ParseConfig("root-config --cflags --ldflags --libs")
 rootenv.Append(LIBS='-lSpectrum')
-rootenv.Append(LIBS=['-lFoam', '-lPyROOT'])
+rootenv.Append(LIBS=['-lFoam', '-lPyROOT','-lMinuit2'])
```

Comment:
Actually I would propose to also add some check at build time to see if the library is available and fail graciously if it didn't. 

Something similar is done in GEANT4.scons to see if the internal CLHEP library should be used.